.github/dependabot.yml

```
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: daily
```

Set Dependabot schedule interval to weekly, so that dependencies are collected and updated once a week and not every day.
```suggestion
      interval: weekly
```